recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb

```diff
@@ -143,10 +137,14 @@ RDEPENDS_packagegroup-wpewebkit-depends-core = "\
     libgcrypt \
     bubblewrap \
     xdg-dbus-proxy \
-    libseccomp \
+    ${@bb.utils.contains('BBFILE_COLLECTIONS', 'security', 'libseccomp', '', d)} \
     libicudata \
 "
-
+RDEPENDS_packagegroup-wpewebkit-depends-core_append_libc-glibc = "\
```

Comment:
Is not necessary to add in the `RDEPENDS_packagegroup-wpewebkit-depends` ?

Reply:
nevermind, I read badly the this line and I didn't realize about the `append_libc-glibc `. The PR LGTM and I already tested the generated SDK using this.


Thanks for the contribution.